### PR TITLE
Feature userinterface

### PR DESCRIPTION
**Pull request by djshatterz**
All Users should be loaded now into one ListView for testing. Displaychanges can be made when users get loaded correctly.

### CODE REVIEW

File: ZbW_P_Contact_Manager/UI/frmAdministration.cs [hunk 1]
```diff
@@ -1,40 +1,193 @@
...
+            MessageBox.Show("Loading users...");
+            listView1.Items.Clear();
+
+            // Load all employees, trainees, and customers
+            List<Person> employees = _userController.Read(ModelType.Employee);
+            List<Person> trainees = _userController.Read(ModelType.Trainee);
+            List<Person> customers = _userController.Read(ModelType.Customer);
+
+            // Add employees
+            foreach (Employee employee in employees.OfType<Employee>())
```
Comment (FabianDuenki) [inline ZbW_P_Contact_Manager/UI/frmAdministration.cs:83]: I think the Method "OfType" is not required. just using "in employees" would make it a little easier to understand

File: ZbW_P_Contact_Manager/UI/frmAdministration.cs [hunk 2]
```diff
@@ -1,40 +1,193 @@
...
+            foreach (Customer customer in customers.OfType<Customer>())
+            {
+                AddUserToListView(customer);
+            }
         }
 
+        // Helper method to add users to the ListView
+        private void AddUserToListView(Person person)
+        {
+            ListViewItem item = new ListViewItem(person.Id.ToString());
```
Comment (FabianDuenki) [inline ZbW_P_Contact_Manager/UI/frmAdministration.cs:104]: that's neat :O love it!

File: ZbW_P_Contact_Manager/UI/frmAdministration.cs (hunk 1 above)
Comment (beyluta) [reply ZbW_P_Contact_Manager/UI/frmAdministration.cs:83]: > I think the Method "OfType" is not required. just using "in employees" would make it a little easier to understand

Yes. Definitely not needed.

File: ZbW_P_Contact_Manager/UI/frmAdministration.cs [hunk 3]
```diff
@@ -1,40 +1,193 @@
...
+            listView1.Columns.Add("Company Name", 150);
+            listView1.Columns.Add("Company Type", 150);
+            listView1.Columns.Add("Company Contact", 150);
+
+            listView1.FullRowSelect = true;
+        }
+
+        private void LoadUsersIntoListView()
+        {
+            MessageBox.Show("Loading users...");
```
Comment (beyluta) [inline ZbW_P_Contact_Manager/UI/frmAdministration.cs:74]: This log is not needed.

**Review by FabianDuenki — APPROVED** [review]

**Pull request merged**